src/api/__specs__/campaigns-spec.js

```diff
@@ -85,7 +85,11 @@ describe('campaigns', () => {
 
     describe('with empty array', () => {
       let _ = require('lodash');
-      let lodashStub = sinon.stub(_, "defer", () => {});
+      let lodashStub;
+
+      before(() => {
+        lodashStub = sinon.stub(_, "defer", () => {});
```

Comment:
Since you are already doing the right thing and restoring this stub you could simplify this code and remove the need for the `lodashStub` variable:

```js
before(() => {
    sinon.stub(_, 'defer', () => {});
});
```

Then for your expectation you could use the `expect` syntax which I feel is more readable and idiomatic:

```js
expect(_.defer).to.be.calledWithExactly(callback, { campaigns: [] });
```

For me I like that this removes the indirection introduced by the `lodashStub` variable.